### ISSUE TITLE
Install script doesn't recognize arm architecture (raspberry pi)

### ISSUE THREAD

**Issue by moebiusV** [joewalnes/websocketd#295]
When I did a git clone than typed "make", I got this output:

```
pi@raspberrypi:~/websocketd $ make
mkdir -p go-v1.4.linux-386/
rm -f go-v1.4.linux-386/.done
Downloading and unpacking Go 1.4.linux-386 to go-v1.4.linux-386/
wget -q -O - http://golang.org/dl/go1.4.linux-386.tar.gz | tar xzf - --strip-components=1 -C go-v1.4.linux-386/
touch go-v1.4.linux-386/.done
chmod +x go
Created ./go wrapper
mkdir -p go-workspace/src/github.com/joewalnes
ln -s ../../../../ go-workspace/src/github.com/joewalnes/websocketd
./go get ./go-workspace/src/github.com/joewalnes/websocketd
/home/pi/websocketd/go-v1.4.linux-386/bin/go: 1: /home/pi/websocketd/go-v1.4.linux-386/bin/go: Syntax error: word unexpected (expecting ")")
Makefile:23: recipe for target 'websocketd' failed
make: *** [websocketd] Error 2
```

Do you plan to update to go v1.9?

Looking in the Makefile, looks like it only supports i386 and amd64.  But a raspberry pi is an arm:

```
pi@raspberrypi:~/websocketd $ uname -a
Linux raspberrypi 4.4.34-v7+ #930 SMP Wed Nov 23 15:20:41 GMT 2016 armv7l GNU/Linux
```

**Comment by asergeyev** [comment]
That script is not really for using when you want to compile binary for yourself.

I recommend to just install go, then say "go get github.com/joewalnes/websocketd"  and expect websocket to be installed in your gopath's bin directory.

You can use any go version that way.

Binaries in github.com/joewalnes/websocketd/releases are compiled with 1.9.2

**Comment by Charlieglider** [comment]
Hi when I tried using` go get` it worked initially, and I was able to run the `websocketd` command, but when I restarted my device (RPI 3B) it wouldn't detect the command anymore. I have to run the `go get ` command every time the device restarts, and I've already set my gopath & gobin as per [https://stackoverflow.com/questions/30295146/how-can-i-install-a-package-with-go-get](url).

Is there a fix for this? Thanks.
